crowdnotifier-sdk/src/main/proto/backend.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package org.crowdnotifier.android.sdk.model;
+
+message QRCodeTrace {
+  uint32 version = 1;
+  MasterTrace masterTraceRecord = 2;
+  bytes notificationKey = 3;
+}
+
+message MasterTrace {
+  bytes masterPublicKey = 1;
+  bytes masterSecretKeyLocation = 2;
+  bytes info = 3;
+  bytes nonce1 = 4;
+  bytes nonce2 = 5;
+  bytes cipherTextHealthAuthority = 6;
+}
+
+message PreTrace {
+  bytes identity = 1;
+  bytes partialSecretKeyForIdentityOfLocation = 2;
+  bytes cipherTextHealthAuthority = 3;
+  string message = 4;
+  bytes notificationKey = 5;
+}
+
+message TraceProof {
+  bytes masterPublicKey = 1;
+  bytes nonce1 = 2;
+  bytes nonce2 = 3;
+}
+
+message PreTraceWithProof {
+  PreTrace preTrace = 1;
+  TraceProof proof = 2;
+  bytes info = 3;
+  int64 startTime = 4;
```

Comment:
Oh my - I currently don't have that in the `libcrowdnotifier` - but it seems to be needed. Shall I add it?

Reply:
Hmm yes, for consistency reasons that would be great!